data/fields/ref/EU/EVSE-EU.json

```
@@ -0,0 +1,18 @@
+{
+    "key": "ref:EU:EVSE",
+    "type": "identifier",
+    "label": "Electrical Vehicle Supply Equipment (EVSE)",
+    "placeholder": "DE*ABC*E12345ABCDEFGHIJKLMNOPQRSTUVWXYZ",
+    "locationSet": {
+        "include": [
+            "EU"
+        ]
+    },
+    "terms": [
+        "electrical vehicle supply equipment",
+        "evse",
+        "charging identifier",
+        "charger identifier"

```

Please sort these terms alphabetically:

```suggestion
        "charger identifier",
        "charging identifier",
        "electrical vehicle supply equipment",
        "evse"
```